docs/quickstart.md

```diff
@@ -0,0 +1,51 @@
+# FastTrack Quickstart
+
+## Install Dependencies
+
+FastTrack requires the following dependencies to be installed on your system:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Python 3](https://www.python.org/downloads/)
+
+```
+
+## Run FastTrack
+
+FastTrack can be run using the following command:
+
+```bash
+docker run --rm -p 5000:5000 -ti gresearch/fasttrack
+```
+
+Verify that you can see the UI by navigating to http://localhost:5000/.
+
+![FastTrack UI](https://files.mcaq.me/57b05.jpg)
+
+
+## Run a quick test script
+
+```bash
+python3 ./docs/dev/minimal.py
+```
+
+After running this script, you should see the following output from http://localhost:5000/aim/:
+
+![FastTrack UI](https://files.mcaq.me/43x5j.jpg)
+
+From here you can check out the metrics and run information to see more details about the run.
+
+## Testing a Random Forrest Model
+
+**Note that since artifacts are not yet supported, most of the autolog features will not work.**
+
+```bash
+# Install mflow and poetry
+cd docs/dev
+poetry install
+# MLFlow will not be installed by poetry, so we need to install it manually
```

Comment:
Why is it not installed by poetry?

Reply:
It broke on my system, and I can't find any refence to people doing it successfully